Hunting Queries/CloudAppEvents/SetPolicyConfigInCloudAppEvents.yaml

```diff
@@ -0,0 +1,32 @@
+id: fcd4d774-a0c2-4d12-9e9f-f51dfc310873
+name: 

```

Comment:
Name is missing.

Reply:
CloudApp Policy Configuration Changes (use copilot)